.github/workflows/cargo_flake.yml

```diff
@@ -33,4 +36,4 @@ jobs:
       - run: echo "::add-matcher::.github/matchers/rust.json"
       - run: cargo install cargo-flake
       - run: make slim-builds
-      - run: cargo flake --features 'default benches metrics-benches remap-benches'
+      - run: cargo flake --features $${ matrix.target }}
```

Comment:
This seems to have failed: https://github.com/timberio/vector/runs/2766459903?check_suite_focus=true

Reply:
I've gone ahead and temporarily modified the PR in https://github.com/timberio/vector/pull/7776/commits/4924152abaa771df377d43db264528e557469098 to run on pull_requests to get faster feedback.